### ISSUE TITLE
Hide Recently Installed and Settings

### ISSUE THREAD

**Issue by DitriSenapati1** [8bitPit/Niagara-Issues#1920]
### Is your feature request related to a problem?

After the new spring update it just feels unlike Niagara to not provide a way to disable the Recently Installed pop up.
For users who need a more cleaner setup I think there should be a way to hide it from the launcher; also, I think there should also be a toggle to hide Niagara settings as it can be easily assessed through long pressing any apps or the widget.
I think it would feel more like Niagara to provide a way to users to declutter their applist even more.
And I am sorry if this already exists because I can not find it anywhere.

### Describe the solution you'd like

Probably a toggle in the Look section for Settings and one in Features section for Recently Installed should solve this.

### Feature Request Procedure

- [X] I have read the [knowledge base article on feature requests](https://help.niagaralauncher.app/article/44-feature-requests) and followed the steps mentioned

**Comment by Foxbat24N** [comment]
Seriously needed this one ... But I don't think they are going to do that... I've talked on this issue in telegram group with @Maxr1998

**Comment by DitriSenapati1** [comment]
Alternatively treating them as Apps or Popup Folders would not only let the users hide them but also Customize them.

**Comment by Oddward** [comment]
Since this is about the fixed shortcuts at the bottom of the app list please refer to issue #1682 and [Peter's comment](https://github.com/8bitPit/Niagara-Issues/issues/1682#issuecomment-924843713) 

The shortcuts are pretty tucked away so they don't interfere with your setup while making it easier for users to find those features (a look at store reviews & stats shows that there are always people that miss things that we think are obvious), many users will naturally look for a settings button, and it can be faster than long-pressing 

.
Customization was planned - refer to issue #1595 - but not sure of its status now

**Comment by DitriSenapati1** [comment]
Yes that may be true about settings but what about the Recently Installed Popup folder, it wasn't there from the beginning so as with all the features that has been added, the user gets to choose wether they want them or not but it is not the case with this specific popup. It just randomly appeared even though I did not wanted and considering that this feature was added anybody could just roll back one version to fix it.

**Comment by Oddward** [comment]
I've already linked the issues for customization and hiding Niagara settings, and the Recently installed pop-up is grouped with it now, it's under the Niagara Launcher section which is a functional part of the UI. It's a digital wellbeing feature to remind users not to keep unused apps around. If it sometimes disappears users will keep requesting to add this feature as they already did before.

It's at the bottom of the list which isn't in anyone's way so it's not considered to be negatively affecting the UX of users that don't need it. The app list isn't part of your setup, it's the launcher's app drawer. It might only be troublesome when the user has no apps in the app list for some reason 😅 

The [design principles](https://help.niagaralauncher.app/article/8-niagaras-design-principles) don't state that all new features must be optional, it states new features must try to meet these criteria:

- **Simple**: It is easy to configure and easy to use. As few settings & toggles as possible.
- **Impactful**:  It is either appealing for many or necessary for some.
- **Universal**:  It has to work everywhere: in every country, on all phones.
- **Independent**: It doesn’t add complexity if it’s not being used.

Most features are designed to work without configuration from the user except where necessary, and most toggle requests go against those guidelines unfortunately.
